common/lib/accuracy.F90

```diff
@@ -8,14 +8,22 @@ module common_accuracy
   implicit none
   private
 
-  public :: dp, cp, sc, mc, lc
+  public :: dp, cp, r8, r4, sc, mc, lc
 
   !> precision of the real data type
   integer, parameter :: dp = real64
 
   !> precision of the complex data type
   integer, parameter :: cp = dp
 
+  !> parameter for real with 8 byte
+  !! (use those only for interfacing F77 routines)
+  integer, parameter :: r8 = 8
+
+  !> parameter for real with 4 byte
+  !! (use those only for interfacing F77 routines)
+  integer, parameter :: r4 = 4
```

Comment:
```suggestion
  integer, parameter :: r4 = kind(1.0)
```

Reply:
Thanks for pointing out, I've applied the suggestion in the latest commit.